src/publish.js

```diff
@@ -13,7 +13,6 @@
 'use strict';
 
 const yargsOpenwhisk = require('./yargs-openwhisk.js');
-const yargsFastly = require('./yargs-fastly.js');
```

Comment:
I suggest to rename the option in `yargs-fastly.js`, so it is consistent for all commands that need fastly credentials.

Reply:
I slacked you about this! :)

Reply:
this command doesn't need fastly-namespace
if we use yargs-fastly it'll attach fastly-namespace as an option

but Lars wants fastly-namespace to be an alias for fastly-serviceid

There will be a collision....fastly-namespace will be both an option and an alias

Reply:
don't even know if that's possible. But it can lead to things like; someone can pass both --fastly-serviceid and --fastly-namespace and if both are allowed to exist... --fastly-namespace can overwrite

Reply:
no, `fastly-namespace` was an unfortunate naming of the parameter for the `serviceid`.
so we should use the `fastly-serviceid` parameter instead.

this parameter is used in `publish.js` and in `deploy.js`. so we should keep it consistent.
you can add an alias using https://github.com/yargs/yargs/blob/master/docs/api.md#aliaskey-alias

eg: 

`alias: 'fastly-namespace',`